crates/daphne-service-utils/src/lib.rs

```diff
@@ -32,3 +34,25 @@ mod durable_request_capnp {
         "/src/durable_requests/durable_request_capnp.rs"
     ));
 }
+
+#[cfg(feature = "durable_requests")]
+mod aggregation_job_store_capnp {
+    #![allow(dead_code)]
+    #![allow(clippy::pedantic)]
+    #![allow(clippy::needless_lifetimes)]
```

Comment:
Can you explain why these warnings need to be suppressed? Here and below

Reply:
There's not point having warnings for generated code